src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryPaths.java

```diff
@@ -6,11 +6,10 @@ private MercuryPaths() {}
 
   // Paths in Mercury Curated Main Pages
   public static final String ROOT_PATH = "/";
+  public static final String ROOT_MAIN_EDIT = "main/edit";
   public static final String ROOT_PATH_SECTION = "/main/section/";
   public static final String ROOT_PATH_CATEGORY = "/main/category/";
   public static final String ROOT_ARTICLE_PATH = "/wiki/";
   public static final String ROOT_BLOG_PATH = "/wiki/User_blog:";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)